addons/web/static/src/core/commands/command_palette.js

```diff
@@ -245,9 +249,14 @@ export class CommandPalette extends Component {
         scrollTo(command, { scrollable: this.listboxRef.el });
     }
 
-    onCommandClicked(index) {
+    onCommandClicked(event, index) {
```

Comment:
Can you add a small comment before the event.preventDefault ?

Something like:

// need to preventDefault as some commands are links with a href, and we want to prevent the redirect